client/me/help/controller.js

```diff
@@ -17,8 +17,23 @@ import { renderWithReduxStore } from 'lib/react-helpers';
 import HelpComponent from './main';
 import CoursesComponent from './help-courses';
 import ContactComponent from './help-contact';
+import userUtils from 'lib/user/utils';
+import support from 'lib/url/support';
 
 export default {
+	loggedOut( context, next ) {
+		if ( userUtils.isLoggedIn() ) {
+			return next();
+		}
+
+		if ( context.path === '/help' ) {
+			window.location.href = support.SUPPORT_ROOT;
```

Comment:
It has limitations with external urls (I guess when running in WP.com context it should be alright). But at the very least we should use the [existing utility for external redirects](https://github.com/Automattic/wp-calypso/blob/master/client/lib/route/path.js#L132-L134).